docs/ZFS-RAID-CONFIGURATION.md

```diff
@@ -0,0 +1,30 @@
+# EVE-OS supported RAID configuration(s)
+
+| RAID Level | Separate EVE-OS disk | Minimum number of disks for RAID | Example Grub Configuration              |
+| ---------- | -------------------- | -------------------------------- | --------------------------------------- |
+| none(raid0)| Not mandatory        |  1                               | eve_install_zfs_with_raid_level = none
+|            |                      |                                  | Single disk config:
+|            |                      |                                  | eve_install_disk=sda
+|            |                      |                                  |
+|            |                      |                                  | Multiple disk config:
```

Comment:
Might make sense to separate out the single and multiple disks thus to separate rows with none(raid0) in the left column. That would make it easier to read.

Reply:
raid0 is just striping, there is no redundancy.  none is value passed to the config parameter eve_install_zfs_with_raid_level, which implies that disks are not RAID protected.

Reply:
But I agree may be its not reasonable to mention none under raid level. I will fix that.